src/tbb/arena.h

```diff
@@ -353,6 +356,11 @@ class arena: public padded<arena_base>
         return num_workers_active() > my_num_workers_allotted.load(std::memory_order_relaxed);
     }
 
+    //! Check that numa optimization works for this arena
+    bool has_numa_optimization() const noexcept {
+        return info::numa_nodes().size() > 1 && static_cast<int>(my_num_slots) == info::default_concurrency() && my_num_reserved_slots == 1u;
```

Comment:
Instead info::default_concurrency you should use governor::default_num_threads()